website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "琚致远：95 后 Apache Member 的成长之路"
+authors:
+  - name: "饶雅惠"
+    title: "Author"
+    url: "https://github.com/EstelleRao"
+    image_url: "https://github.com/EstelleRao.png"
+keywords:
+- Apache APISIX
+- API 网关
+- ASF Member
+- Apache Member
+- 开源
+- 远程
+- web 全栈工程师
+description: 2022 年 3 月，Apache APISIX PPMC 成员琚致远（GitHub ID：juzhiyuan）入选 Apache Member 和中国开源码力榜。Apache APISIX 社区对此表示祝贺，也邀请了致远做客访谈，聊聊他的开源经历和技术成长之路。
+tags: [Interview]
+---
+
+> 2022 年 3 月，Apache APISIX PPMC 成员琚致远（GitHub ID：juzhiyuan）入选 Apache Member 和中国开源码力榜。Apache APISIX 社区对此  
+表示祝贺，也邀请了致远做客访谈，聊聊他的开源经历和技术成长之路。
+
+<!--truncate-->
+
+## 导语
+
+**琚致远（GitHub ID：juzhiyuan）于 1997 年出生，2019 年 10 月成为 Apache APISIX PPMC Member，2022 年 3 月入选 Apache Member 和  
+中国开源码力榜。他是开源软件商业公司 API7.ai 的 Global Team Leader，也是 ApacheCon Speaker、OSPP Mentor、GSoC Mentor、（2019 年之前）freeCodeCamp 中国核心组织者之一。**
+
+![ASF Members](https://static.apiseven.com/202108/1651198214277-8bb73696-dbe0-4e24-b0b9-2bb65e1abbfd.png)
+
+他小学开始接触电脑，学习 Flash 技术；初高中自学 Web 渗透，爱上黑客文化，向往成为一名乌云白帽；大学加入 freeCodeCamp 社区，成为 Web 技术的  
+布道师、引路人，从此和开源结下不解之缘。毕业后，他成为开源社区的深度参与者和 Apache APISIX PMC 成员，广泛参与活动，对外讲述自己的开源经历、  
+技术成长经历，传播开源文化和 Apache 文化。
+
+**他说他是个极客，敢想敢做，不被条条框框束缚，所做的一切只是因为热爱，因为追求真理。他说每个开源项目的背后都有一个开源社区，开源精神在于我们有想法，  
+我们想实现出来，公开出来，邀请更多的人参与进来。他说他很开心在技术学习、开源参与这条路上能为后来人提供力所能及的帮助，自己成长的前提是身边的朋友一起成长。**
```

Comment:
ditto

Reply:
Noted. Will be fixed together with the issue brought up by yzeng25.